mypy/build.py

```diff
@@ -249,10 +257,13 @@ def mypy_path() -> List[str]:
     return path_env.split(os.pathsep)
 
 
-def default_lib_path(data_dir: str, pyversion: Tuple[int, int]) -> List[str]:
+def default_lib_path(data_dir: str,
+                     pyversion: Tuple[int, int],
+                     custom_typeshed_name: str = None) -> List[str]:
     """Return default standard library search paths."""
     # IDEA: Make this more portable.
     path = []  # type: List[str]
+    typeshed_name = custom_typeshed_name or 'typeshed'
 
     auto = os.path.join(data_dir, 'stubs-auto')
```

Comment:
What's going on here?  What is `stubs-auto`?  (The commit that introduces it doesn't explain at all...)